CONTRIBUTING.md

```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
 
 # Building Operation Code
 
-Building Operation Code is quite simple.
-
-```sh
-cd operationcode
-gem install bundler
-bundle
-bundle exec rake db:setup
-bundle exec rails server
-```
+## Mac OS X
+1. install homebrew ( http://brew.sh/ - package manager for OS X)  
+  * If you use macports, you might consider changing to homebrew. 
+  * This step is optional, but will make your life easier.
+2. install postgres if you dont have it already
+  1. ```brew install postgres```
+  2. start postgres ( follow brew directions after install )
+  3. ```createuser davidmolina -d```
+3. install bundler ( http://bundler.io )
+4. Install rbenv ( via homebrew or github, https://github.com/sstephenson/rbenv )
+  1. ```brew update```
+  2. ```brew install rbenv ruby-build```
```

Comment:
Single backpacks and we should probably not harp on what package manager users use. I myself don't use rbenv or rvm.